### PR TITLE
Login System

### PR DESCRIPTION
**Pull request by Hannah-Hoover**
Completed the login system

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -218,7 +218,7 @@ button {
     overflow: hidden; /* Hide the overflow text */
     white-space: nowrap; /* Prevent text from wrapping */
     animation: typing 4s steps(40, end) 1s 1 normal both; /* 1 iteration */
-    color: #8B4513; /* Start with a brown color (adjust to your desired shade) */
```
Comment (Naz1380) [inline style.css:221]: does this cause redundancy?

File: style.css [hunk 2]
```diff
@@ -218,7 +218,7 @@ button {
     overflow: hidden; /* Hide the overflow text */
     white-space: nowrap; /* Prevent text from wrapping */
```
Comment (Naz1380) [inline style.css:203]: overall, great job for debugging the issue with the button!

**Review by Naz1380 — APPROVED** [review]

**Review by Lavender-C — APPROVED** [review]
As far as I can tell it looks good, maybe add some comments here and there to describe what's going on? (Not a priority RN, if you have some free time)

Thank you for your hard work.

**Review by MohimaKibria — APPROVED** [review]
The code is well organized and easy to follow. Looks good to me!

**Pull request merged**